packages/frontier-plugins/plugin-qa/README.md

```diff
@@ -1,41 +1,81 @@
-# rdspec
+<div align="left">
+  <br/>
+  <a href="https://www.realdecoy.com/jamaica/" title="REALDECOY">
+    <img width=400px src="https://www.realdecoy.com/wp-content/uploads/2019/02/Realdecoy-logo-transparent.png" alt="rd logo">
+  </a>
+  <br/>
+</div>
 
+
+# Spectre
+
+### @rdfrontier/plugin-qa
 CLI to generate web automation projects
 
 [![oclif](https://img.shields.io/badge/cli-oclif-brightgreen.svg)](https://oclif.io)
 [![Version](https://img.shields.io/npm/v/rdspec.svg)](https://npmjs.org/package/rdspec)
 [![Downloads/week](https://img.shields.io/npm/dw/rdspec.svg)](https://npmjs.org/package/rdspec)
```

Comment:
update link to https://npmjs.org/package/@rdfrontier/plugin-qa

Reply:
replicate in plugin-vue

Reply:
updated